Simplified/Audiobooks/AudiobookManifestAdapter.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AudiobookManifestHelper.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2022-03-30.
+//  Copyright © 2022 NYPL. All rights reserved.
+//
+#if FEATURE_AUDIOBOOKS
+
+import Foundation
+import NYPLAudiobookToolkit
+import UIKit
+
+#if FEATURE_OVERDRIVE_AUTH
+import OverdriveProcessor
+#endif
+
+@objc enum AudiobookManifestError: Int {
+  case none
+  case unsupported
+  case corrupted
+}
+
+@objc class AudiobookManifestAdapter: NSObject {
+  @objc class func transformAudiobookManifest(book: NYPLBook,
```

Comment:
can you add some docs about what kind of transformation we're doing here? It seems like we are taking the audiobook manifest file and converting that to a Dictionary? If so I'd rename the method to something more direct, like `transformManifestToDictionary(forAudiobook:completion...)`